.lando.yml

```diff
@@ -0,0 +1,45 @@
+name: drupal-orange-project
+recipe: drupal8
+config:
+  webroot: web
+  xdebug: true
+
+services:
+  index:
+    # Use solr version 7.1
+    type: solr:7.1
```

Comment:
If there isn't a specific requirement to use 7.1, we might want to leave it as the default (by specifying `type: solr`. If there is, such as if search_api_solr only supports up to 7.1, let's change the comment to say why we want 7.1. That way if that circumstance changes we know whether this can be updated or not. As is I'd be uncomfortable ever changing this version since I don't know why it's set there in the first place.

Reply:
Ah, 7.1 is the default. https://docs.devwithlando.io/services/solr.html